.circleci/config.yml

```diff
@@ -62,12 +67,19 @@ workflows:
           requires:
             - build
 
+      - e2eTestChartInstallMaster:
```

Comment:
We should just simplify here. chart-operator has e2e jobs with `PR` and `Master` suffixes. That is because of the publishing to the stable release channel. I don't think this is needed.